### PR TITLE
add us keyboard hint to index site

### PR DESCRIPTION
**Pull request by chrbeckm**
Closes last point of #277

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -54,23 +54,22 @@ bis auf [VSCode](/install/windows_10/#VSCode) (kostenlos, aber proprietär).
...
+  Wir bringen euch die Grundlagen bei.
+
 * **Git:** Reproduzierbarkeit und kollaboratives Arbeiten sind zwei Hauptaspekte
   des guten wissenschaftlichen Arbeitens.
   Git ist das Standardwerkzeug für Versionskontrolle,
   welche es ermöglicht jederzeit zu älteren Versionen zurückzukehren,
   Änderungen nachzuvollziehen und gemeinsam an Projekten zu arbeiten.
 
-* **Make**: Das Standardtool zur automatisierten Ausführung von komplexeren Aufgaben.
+  * **Make**: Das Standardtool zur automatisierten Ausführung von komplexeren Aufgaben.
```
Comment (SepplL) [inline index.md:67]: Maybe rephrase to:
- Eins der standard-Tools zur ...
- Unser favorisiertes standard-tool zur ...

**Review by SepplL — APPROVED** [review]

**Review by aknierim — APPROVED** [review]
